components/VFBMain.js

```diff
@@ -429,10 +427,7 @@ export default class VFBMain extends React.Component {
 
   UIUpdateItem (itemState, visibilityAnchor) {
     if (this.state[itemState] === false) {
-      this.setState({
-        UIUpdated: true,
```

Comment:
this should make it back since it's used for the ShouldComponentUpdate to minimise the not needed re-render